include/git2cpp/blame.h

```diff
@@ -1,23 +1,25 @@
-﻿#pragma once
+#pragma once
 
-#include <memory>
+#include <git2/blame.h>
 
-struct git_blame;
-struct git_blame_hunk;
+#include <memory>
 
 namespace git
 {
     struct Blame
     {
+        uint32_t hunk_count() const;
+
+        const git_blame_hunk * hunk_byindex(uint32_t index) const;
+
+        const git_blame_hunk * hunk_byline(size_t lineno) const;
+
         explicit Blame(git_blame * blame)
             : blame_(blame)
         {}
 
-        git_blame_hunk const * get_hunk_byline(size_t lineno) const;
-
     private:
         struct Destroy { void operator() (git_blame *) const; };
-
-        std::unique_ptr<git_blame, Destroy> blame_;        
+        std::unique_ptr<git_blame, Destroy> blame_;
```

Comment:
This change is redundant.

Reply:
This just removes the empty line, which is consistent with diff.h